src/mpyl/cli/commands/projects/lint.py

```diff
@@ -0,0 +1,157 @@
+"""Helper methods for linting projects for correctnessare can be found here"""
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+
+import click
+import jsonschema
+from rich.console import Console
+
+from ....steps.deploy.k8s import substitute_namespaces
+from ....steps.deploy.k8s.chart import ChartBuilder
+from ....utilities.repo import Repository
+from ....cli.commands.build.mpyl import find_build_set
+from ....project import Project, load_project, ProjectName, Target
+
+
+def _find_projects(all_: bool, repo: Repository, filter_: str):
```

Comment:
Hm, but it finds changed projects and I feel like we've needed that before elsewhere. `.path` is just part of `Project` so that's not really relevant for this discussion I guess

Reply:
We do similar-ish things in the main mpyl flow, and when we do `mpyl build status`. There's not much to refactor though, because the changes are different per place, so you would basically only write a wrapper around find_build_set, because also its result is used differently at those places